### DISCUSSION TITLE
IPTV App para iphone

### DISCUSSION THREAD

**Discussion by LUISSM3** [LITUATUI/M3UPT#19] [Q&A]
Antes de tudo o meu agradecimento por este excelente projeto. É o único que encontrei que mantém a Bloomberg a funcionar de forma estável. Obrigado.

Não sei se é possível encontrar alguma fonte de streaming da CNBC e a FOX BUSINESS para associar à lista.
Já agora, qual a aplicação que recomendas para utilizar nos iphones?

No PC uso a aplicação MyIPTV Player (ao contrário do Kodi, permite continuar a utilizar o pc e manter apenas uma janela com o canal aberto) https://www.microsoft.com/pt-pt/p/myiptv-player/9pjj2nmbf0tr#activetab=pivot:overviewtab
Nas TV's LG WebOs e nas TV's Samsung Tizen utilizo o OttPlayer https://ottplayer.tv/
No Android também utilizo o OttPlayer.
No IOS ou IOSPAD ainda não decidi qual a aplicação utilizar. O que recomendas?
Obrigado.

**Comment by LITUATUI** [comment]
Olá.

Esses dois canais não disponibilizam os streams nos seus sites.

No PC podes ter o Kodi aberto numa janela com o tamanho que quiseres, basta carregares na tecla \ e o Kodi passa de ecrã completo (full screen) para modo janela (windowed mode).

Para iOS não há nenhuma aplicação que recomende, pois a minha experiência como utilizador resume-se a Linux no portátil e Android no telemóvel. Pode ser que algum utilizador desse sistema operativo veja este tópico e possa fazer uma recomendação.

**Reply by LUISSM3** [reply]
Obrigado pela ajuda. Já agora, utilizo dois monitores, mas não consigo que o Kodi funcione no segundo monitor, sabes como ultrapassar isso?

**Reply by LITUATUI** [reply]
Vê se [isto](https://forum.kodi.tv/showthread.php?tid=330437) ajuda.

**Reply by LUISSM3** [reply]
Boa :) ajuda sim, já resolvi o problema, obrigado.

**Comment by JMJorge** [comment]
Funciona com gse IPTV, apesar dos anúncios serem um pouco chatos
